galadriel/cli.py

```diff
@@ -241,57 +238,63 @@ def import_wallet(private_key: str, path: str):
+            keypair = Keypair.from_bytes(keypair_bytes)
+            pubkey = str(keypair.pubkey())
+
+            # Save to default path
+            os.makedirs(os.path.dirname(DEFAULT_SOLANA_KEY_PATH), exist_ok=True)
+            with open(DEFAULT_SOLANA_KEY_PATH, "w") as f:
+                f.write(private_key)
+
+            click.echo(f"Successfully imported wallet {pubkey} to {DEFAULT_SOLANA_KEY_PATH}")
+        else:
+            # Import from file
+            with open(path, "r") as f:
+                keypair_json = f.read()
 
-    if private_key:
-        # Check if the private key is a valid json
-        try:
-            json.loads(private_key)
-        except json.JSONDecodeError:
-            raise click.ClickException("Invalid private key! Please provide a valid JSON array")
-        # Save the private key to the default path
-        os.makedirs(os.path.dirname(DEFAULT_SOLANA_KEY_PATH), exist_ok=True)
-        with open(DEFAULT_SOLANA_KEY_PATH, "w", encoding="utf-8") as file:
-            file.write(private_key)
-        _update_agent_env_file({"SOLANA_KEY_PATH": DEFAULT_SOLANA_KEY_PATH})
+            keypair_bytes = bytes(json.loads(keypair_json))
+            keypair = Keypair.from_bytes(keypair_bytes)
+            pubkey = str(keypair.pubkey())
 
-        click.echo("Successfully imported Solana wallet from private key")
-    elif path:
-        if not os.path.exists(path):
-            raise click.ClickException(f"File {path} does not exist")
-        _update_agent_env_file({"SOLANA_KEY_PATH": path})
+            # Copy to default path
+            os.makedirs(os.path.dirname(DEFAULT_SOLANA_KEY_PATH), exist_ok=True)
+            shutil.copy(path, DEFAULT_SOLANA_KEY_PATH)
 
-        click.echo(f"Successfully imported Solana wallet from {path}")
+            click.echo(f"Successfully imported wallet {pubkey} from {path} to {DEFAULT_SOLANA_KEY_PATH}")
+    except Exception as e:
+        click.echo(f"Failed to import wallet: {str(e)}")
 
 
 @wallet.command()
 def airdrop():
-    """Request an airdrop of 0.001 SOL to the given Solana wallet."""
-
-    load_dotenv(dotenv_path=Path(".") / ".agents.env", override=True)
-
-    key_path = os.getenv("SOLANA_KEY_PATH")
-    if not key_path or not os.path.exists(key_path):
-        raise click.ClickException(
-            "SOLANA_KEY_PATH not found in environment or does not exist. Please run `galadriel wallet create` to create a new wallet or `galadriel wallet import` to import an existing wallet."
-        )
-
+    """Request an airdrop of SOL to the default wallet"""
     try:
-        with open(key_path, "r", encoding="utf-8") as file:
-            seed = json.load(file)
-            pub_key = Keypair.from_bytes(seed).pubkey()
-            _request_airdrop(str(pub_key))
-    except json.JSONDecodeError:
-        raise click.ClickException(f"Invalid JSON format in key file: {key_path}")
+        # Check if wallet exists
+        if not os.path.exists(DEFAULT_SOLANA_KEY_PATH):
```

Comment:
seems here it's hardcoded to use DEFAULT_SOLANA_KEY_PATH, which will make a customized path not working, isn't it?